moshi/src/main/java/com/squareup/moshi/EnumJsonAdapter.java

```diff
@@ -0,0 +1,48 @@
+package com.squareup.moshi;
+
+import java.io.IOException;
+import java.lang.reflect.AnnotatedElement;
+import java.lang.reflect.Type;
+
+/**
+ * Convert Enum types to JSON string values. This uses the enum class's
+ * <pre>toString()</pre> to allow for customization.
```

Comment:
Hmm... I'm not sure `toString()` is the method we'd want to use for this (vs. just `name()`). #29 and/or registering a custom `JsonAdapter` would allow for customization.